### PR TITLE
feat: #GH73 make DoE and participant shown as subnodes

### PR DESCRIPTION
**Pull request by SimonWoodRhea**
### Prerequisites

- [x] I have written a descriptive pull-request title
- [x] I have verified that there are no overlapping [pull-requests](https://github.com/RHEAGROUP/COMET-IME-Community-Edition/pulls) open
- [x] I have verified that I am following the COMET-IME [code style guidelines](https://raw.githubusercontent.com/RHEAGROUP/COMET-IME-Community-Edition/master/.github/CONTRIBUTING.md)
- [x] I have provided test coverage for my change (where applicable)

### Description
Engineering model layout changes.
Participants now show DoE as child nodes
DoE rows also show participants as child nodes. Edit and depreciate menu options have been removed.

### CODE REVIEW

File: CDP4SiteDirectory/ViewModels/ModelBrowser/Rows/ModelParticipantRowViewModel.cs [hunk 1]
```diff
@@ -94,6 +126,14 @@ private void UpdateProperties()
                 this.RoleName = this.Role.Name;
                 this.RoleShortName = this.Role.ShortName;
             }
+
+            if (showDomains)
```
Comment (alexatrhea) [inline CDP4SiteDirectory/ViewModels/ModelBrowser/Rows/ModelParticipantRowViewModel.cs:130]: `this.` qualifier missing

File: CDP4SiteDirectory/ViewModels/ModelBrowser/Rows/EngineeringModelSetupRowViewModel.cs [hunk 2]
```diff
@@ -178,6 +204,8 @@ private void UpdateContainers()
                 this.AddOrganization(organization);
             }
 
+            UpdateDomainParticipants(this.Thing.Participant);
```
Comment (alexatrhea) [inline CDP4SiteDirectory/ViewModels/ModelBrowser/Rows/EngineeringModelSetupRowViewModel.cs:207]: `this.` qualifier missing

**Review by alexatrhea — CHANGES_REQUESTED** [review]
very small things noticed.

**Review by lxatrhea — APPROVED** [review]

**Review by alexatrhea — APPROVED** [review]

**Comment by MartinFijneman** [comment]
this issue was tested for 9.0.0 RC1, working as expected, with the exception of updating as reported in separate issue #932

**Pull request merged**